leshan-integration-tests/src/test/java/org/eclipse/leshan/integration/tests/util/QueueModeLeshanServer.java

```diff
@@ -144,19 +142,21 @@ public void destroy() {
     @Override
     public <T extends LwM2mResponse> T send(final Client destination, final DownlinkRequest<T> request)
             throws InterruptedException {
-        return queueRequestSender.send(destination, request, sendTimeout);
+        throw new UnsupportedOperationException(
+                "This is a deprecated message and better not use it for testing purposes");
```

Comment:
I am not sure what to make of this message
> better not use it for testing purposes

Maybe something along the lines of

*Sender supports asynchronous requests only*

The fact that this method cannot be used is already clear because of the exception, isn't it?